src/konnektor/network_planners/generators/minimal_spanning_tree_network_generator.py

```diff
@@ -88,13 +88,20 @@ def generate_ligand_network(self, components: Iterable[Component]) -> LigandNetw
 
         mg = self.network_generator.generate_network(edges, weights)
 
-        if not mg.connected:
-            nodes_index = {c: components.index(c) for c in components}
-            missing_nodes = [c for c in components if (nodes_index[c] in mg.nodes)]
-            raise RuntimeError("Unable to create edges for some nodes: " + str(list(missing_nodes)))
-
+        # TODO: collect all the mappings, use j->i mapping if i->j not found? - double check this
```

Comment:
I _think_ this make sense, it assumes that the LigandAtomMapping exists but defined by the components in the other way around. Does this make sense to you?